LibreNMS/Alert/Transport/Alerta.php

```diff
@@ -33,22 +32,18 @@ public function deliverAlert(array $alert_data): bool
             'event' => $alert_data['name'],
             'environment' => $this->config['environment'],
             'severity' => $severity,
-            'service' => [$alert_data['title']],
```

Comment:
You aren't sending title anywhere now rather than just removing the duplicate use of it.

One other way to deal with this is allow the templates to merge/override the $data values. I did something similar for the Grafana transport: https://github.com/librenms/librenms/blob/3aeb94232d38c09851975f133e26806b127858c8/LibreNMS/Alert/Transport/Grafana.php